components/widget/match/summary/widget_match_summary_all.lua

```diff
@@ -19,6 +19,7 @@ Widgets.Character = Lua.import('Module:Widget/Match/Summary/Character')
 Widgets.GameCenter = Lua.import('Module:Widget/Match/Summary/GameCenter')
 Widgets.GameComment = Lua.import('Module:Widget/Match/Summary/GameComment')
 Widgets.GameWinLossIndicator = Lua.import('Module:Widget/Match/Summary/GameWinLossIndicator')
+Widgets.MatchComment = Lua.impot('Module:Widget/Match/Summary/MatchComment')
```

Comment:
```suggestion
Widgets.MatchComment = Lua.import('Module:Widget/Match/Summary/MatchComment')
```

Reply:
That's why the LLS was playing tricks on me 😆